recipe/build.sh

```diff
@@ -9,7 +9,26 @@ if [[ "${target_platform}" == "osx-arm64" ]]; then
   export CMAKE_ARGS="${CMAKE_ARGS} -DDUCKDB_PLATFORM=osx_arm64 -DDUCKDB_EXPLICIT_PLATFORM=osx_arm64"
 fi
 
-cmake ${CMAKE_ARGS} -GNinja -DCMAKE_INSTALL_PREFIX=$(pwd)/dist -DOVERRIDE_GIT_DESCRIBE=v$PKG_VERSION-0-g4a89d97 ..
+if [ "${target_platform}" = "linux-64" ]; then
+    DUCKDB_ARCH='linux_amd64'
+elif [ "${target_platform}" = "osx-64" ]; then
+    DUCKDB_ARCH='osx_amd64'
+elif [ "${target_platform}" = "osx-arm64" ]; then
+    DUCKDB_ARCH='osx_arm64'
+else
+    echo "Unknown target platform: ${target_platform}"
+    exit 1
+fi
+
+# Persist DuckDB architecture for extension installation scripts.
+echo "${DUCKDB_ARCH}" > "$(pwd)"/.duckdb_arch
```

Comment:
Please always quote the whole path for platform compatability.
```suggestion
echo "${DUCKDB_ARCH}" > "$(pwd)/.duckdb_arch"
```